packages/server/tests/acceptance/conformityTests.spec.ts

```diff
@@ -699,3 +699,294 @@ describe('@api-conformity @conformity-batch-3 Ethereum execution apis tests', as
     synthesizeWsTestCases(TEST_CASES, updateParamIfNeeded);
   });
 });
+
+describe('@api-conformity @conformity-batch-4 Ethereum execution apis tests', async function () {
+  this.timeout(240 * 1000);
+
+  let existingCallerContractAddress: string;
+  let existingLogsContractAddress: string;
+  let fromBlockForLogs: string;
+
+  before(async () => {
+    const deployCallerContractTx = await signAndSendRawTransaction({
+      chainId: 0x12a,
+      to: null,
+      from: sendAccountAddress,
+      maxPriorityFeePerGas: gasPrice,
+      maxFeePerGas: gasPrice,
+      gasLimit: gasLimit,
+      type: 2,
+      data: CallerContract.bytecode,
+    });
```

Comment:
could be replaced by promise

Reply:
The advantages of replacing with a promise would be minimal.